test/unit/lib/utils.js

```diff
@@ -0,0 +1,29 @@
+const chai = require('chai');
+const utils = require('../../../app/lib/utils');
+
+const expect = chai.expect;
+
+describe('utils', () => {
+  describe('deepClone', () => {
```

Comment:
Nothing derogatory implied by poor man just that it is a partial object clone (in that it just does public properties not methods) which is easy to implement.